specs/dotsigstore_bundle/examples/blob_cert_rekor_timestamps.json

```diff
@@ -0,0 +1,38 @@
+{
+  "mediaType": "application/vnd.dev.sigstore.bundle+json;version=0.1",
+  "timestampVerificationData": {
+    "tlogEntries": [
+      {
+        "logIndex": "123",
+        "logId": "bG9naWQtMTIz",
+        "kindVersion": {
+          "kind": "hashedrecord",
+          "version": "v0.0.1"
```

Comment:
Any reason `kind` and `version` are nested inside an object and not top-level keys? Maybe nit-picking at the generated json but thinking this is what most people will see if they open up the bundle.

Reply:
Grouping them together adds context, as they are both related to a specific entry, in the Rekor response. So when grouped together it's clear that the version refers to the version of the type (kind) and not any version of the response or so.

Reply:
Ah yes this makes sense 👍  

I wonder if this could still be flattened like so?  
```
"kind": "hashedrecord",
"kindVersion": "v0.0.1"
```

I'm assuming this would also be confusing as the other fields are also related to the rekor entry?:

```
...
"logId": "bG9naWQtMTIz",
"rekorEntry": {
  "kind": "hashedrecord",
  "version": "v0.0.1"
```